src/Services/QueuedJobService.php

```diff
@@ -1007,10 +1007,10 @@ public function runJob($jobId)
 
     /**
      * @param QueuedJobDescriptor $jobDescriptor
-     * @param QueuedJob $job
+     * @param QueuedJob|null $job
      * @param Exception|\Throwable $e
      */
-    protected function handleBrokenJobException(QueuedJobDescriptor $jobDescriptor, QueuedJob $job, $e)
+    protected function handleBrokenJobException(QueuedJobDescriptor $jobDescriptor, $job, $e)
```

Comment:
shouldn't this be `?QueuedJob $job` instead?
https://www.php.net/manual/en/migration71.new-features.php

Reply:
Correct, I wasn't sure if I can use that syntax though.